src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -182,6 +182,26 @@ internal static Settings FromDefaultSources()
             return null;
         }
 
+        private static TracesExporter ParseTracesExporter(IConfigurationSource source)
+        {
+            var tracesExporterEnvVar = source.GetString(ConfigurationKeys.TracesExporter) ?? "otlp";
+            switch (tracesExporterEnvVar)
+            {
+                case null:
+                case "":
+                case "otlp":
+                    return TracesExporter.Otlp;
+                case "zipkin":
+                    return TracesExporter.Zipkin;
+                case "jaeger":
```

Comment:
I created an issue as I am not sure if this a proper value: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/355